src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -16,17 +16,20 @@
 import seedu.address.model.person.Email;
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Phone;
+import seedu.address.model.person.Status;
 
 public class JsonAdaptedPersonTest {
     private static final String INVALID_NAME = "R@chel";
     private static final String INVALID_PHONE = "+651234";
     private static final String INVALID_ADDRESS = " ";
     private static final String INVALID_EMAIL = "example.com";
+    private static final String INVALID_STATUS = "active";
```

Comment:
Perhaps you can consider not using "active" here? Probably can use some gibberish like "asdjnewfpqwifnqiweyfbqwekfbrw"?